CorsixTH/Lua/dialogs/menu.lua

```diff
@@ -546,8 +546,14 @@ function UIMenuBar:makeMapeditorMenu(app)
   local menu = UIMenu()
   local hotkeys = app.hotkeys
 
+  local function check_before_save(ui)
```

Comment:
Function name code style is incorrect, lowercase with underscores is for variables.

Also, the name is too detailed. Don't try to add every little step that a function does in the name. Today there is only one additional step but in time more steps could be added. You'll soon run out of line length if you want to express them all, not to mention you need to constantly change the function name throughout the entire program (in theory, this case is simple).  
You have a several lines room in the Luadoc of the function for a more detailed description, or in the code itself in comments.

Also, the name looks too generic. There are several things that can be saved in the program, `save` can be applied on all those things yet the function doesn't handle all those things. It saves maps only.